README.md

```diff
@@ -105,57 +117,129 @@ inv -l
-- `dataset` : datasets listings used in many pages.
-- `reuse` : cards for displaying dataset reused in the real world.
-- `participez` : is the large blue callout seen on multiple pages.
-- `macros` : **TODO**
-- `svg` : contains SVG assets to be included in our pages.
-
-### TODO
-Front docs todo :
-- Parcel 2 architecture
-  - Static copy
-  - Stylemark build
-  - VueJS compiler mode
-- VanillaJS IIFE architecture
-- Vue 3 architecture
-  - Modals
-  - I18n
-  - Config plugin
-  - Components
-Back docs todo :
-- CSS/JS file inclusion
-- Static route for UI-Kit
+- The [Less](https://lesscss.org/), [Vue](https://vuejs.org) & other sourcefiles for the are located in `theme`.
 
-Whenever a components needs some special styling, you can find their corresponding definitions inside `theme/less/specific/<component>`,
-it's best if we can avoid having too much specific styling, but sometimes you just really need it.
+### 🚀 Getting started
+
+Before you start your developper journey, you have to setup your front-end tools.
+
+#### 🏗 Installing the dependencies
+
+First, you need to install [Node](https://nodejs.org/en/download/) (version 14+) on your platform. Then, in the project directory, run :
+
+```shell
+npm install
+```
+
+And voilà ! ✨
+
+#### 💪 Starting the development server
+
+Simply run this command in the project directory :
+
+```shell
+npm start
+```
+
+This will start a development server that will listen to changes and automatically rebuild the project when needed.
+Note that a webserver is started by Parcel (default port is `1234`), however we will not be using it as our CSS and JS files will be served by Jinja instead. More on that later.
+
+#### 👀 Other dev commands
 
 Finally, we have a bunch of commands to make your life a tad easier, that you can run through `npm run`.
-- `build`: Builds the final CSS/JS files and the UI-Kit Documentation. You should probably use this one.
+
+- `build`: Builds the final CSS/JS files and the UI-Kit Documentation. You should probably use this one in production.
 - `build:app`: Builds the final CSS/JS files without the UI-Kit
 - `build:stylemark`: Builds the UI-Kit files and also the CSS/JS files but unminifed (do not use those static files in production)
 - `i18n:report`: Generates a report of the i18n missing and unused keys
 - `i18n:extract`: Same as above, but also automatically adds missing keys to translation files
 - `clean`: Cleans Parcel cache. Use this if you stumble upon weird bugs to start anew.
-- `start`: Get to coding with live reload and things
+- `start`: Get to coding with live reload and things. Same as `npm run dev`
+- `test`: Runs the Cypress tests. More on that in the Tests section of this README.
+
+### 🏰 General architecture
+
+#### 🚜 Jinja2 templates
+
+Because [udata][] is written in Python, its templating engine is Jinja 2.
+This means that the HTML received by clients is built at runtime, for each request, using good ole templates with `{% block %}` tags and includes.
```

Comment:
good ole jinja2 templates with jQuery, just like ye old times 🤠

Reply:
> re: duplicate, I wasn't quite sure which part of the README is still up-to-date for the python part. I guess we could have two main parts : one for the python-related stuff and one for the javascript stuff with a general introduction ?

I think that both `python` and `javacript`-related stuff could be put in the Getting Started section because the two work mostly together. Most information look up-to-date @quaxsze.

Reply:
Sorry :p
```suggestion
This means that the HTML received by clients is built at runtime, for each request, using templates with `{% block %}` tags and includes.
```